sdk/batch/arm-batch/test/batchManagementClient.spec.ts

```diff
@@ -0,0 +1,577 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { BatchManagementClient, BatchManagementModels } from "../src/batchManagementClient";
+import { describe, beforeEach } from "mocha";
+import { assert } from "chai";
+import * as dotenv from "dotenv";
+import * as util from "util";
+import * as fs from "fs";
+import { ServiceClient, WebResource } from "@azure/core-http";
+import { AuthenticationContext } from "adal-node";
+import { TokenCredentials, RestError } from "@azure/ms-rest-js";
+import { CertificateCreateOrUpdateParameters } from "../src/models";
+
+dotenv.config();
+
+describe("Batch Management Service", () => {
+  let client: BatchManagementClient;
+  let subscriptionId: string;
+  let clientId: string;
+  let secret: string;
+  let tenant: string;
+  let location: string;
+  let autoStorage: string;
+  let batchAccount: string;
+  let groupName: string;
+
+  async function getAppOnlyToken(clientId, secret) {
+    const authContext = new AuthenticationContext(
+      "https://login.microsoftonline.com/microsoft.onmicrosoft.com"
+    );
+    return new Promise<string>((resolve, reject) => {
+      authContext.acquireTokenWithClientCredentials(
+        "https://management.core.windows.net/",
+        clientId,
+        secret,
+        (err, token) => {
+          if (err) {
+            reject(err);
+          } else {
+            resolve(token.accessToken);
+          }
+        }
+      );
+    });
+  }
+
+  beforeEach(async () => {
+    subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"]!;
+    location = process.env["AZURE_TEST_LOCATION"]!;
+    autoStorage = process.env["AZURE_AUTOSTORAGE"]!;
+    batchAccount = "batchtestnodesdk";
+    groupName = util.format("default-azurebatch-%s", location);
+    clientId = process.env["AZURE_CLIENT_ID"];
+    secret = process.env["AZURE_CLIENT_SECRET"];
+    tenant = process.env["AZURE_TENANT_ID"];
```

Comment:
would you please add a `sample.env` file similar to this [one](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/batch/batch/test/sample.env) where you explain the role of each variable.